go.mod

```diff
@@ -3,47 +3,27 @@ module github.com/google/ko
 go 1.12
```

Comment:
Can you mark this as generated?

Reply:
Sometimes it's not generated :(

Reply:
That's true, sometimes `go.mod` is meaningful. But `go.sum` and `vendor/modules.txt` is AFAIK always generated, we could just skip those. (Doesn't have to be in this PR)